src/docs/devices/Apollo-Automation-MSR-1/index.md

```diff
@@ -0,0 +1,33 @@
+---
+title: Apollo mmWave CO2 Multisensor (MSR-1)
+date-published: 2023-09-12
+type: sensor
+standard: global
+board: esp32
+project-url: https://github.com/ApolloAutomation/MSR-1/blob/main/Integrations/ESPHome/MSR-1.yaml
+difficulty: 1
+---
+
+The Apollo Automation MSR-1 is a mmWave CO2 multisensor with the following features:
```

Comment:
(same suggestion as above regarding headers for distinct sections for readability/navigation)